scripts/grab-upstream.sh

```diff
@@ -24,6 +24,8 @@ excludes=(
   --exclude=glfw-*/tests
 )
 
+set -e
```

Comment:
Consider moving this up to be at the top of script (on line 3, similar to how it's done in generate-wayland-protocols.sh). The motivation is to follow the pattern of setting it at the very beginning of the script, before doing anything else, and make it easier for code readers to see it's there.

Reply:
While here, you could also add it to `glfw_tree_rebuild.sh`, so that all 3 scripts have it. No reason for that one not to.